common/persistence/class.SqlKvDriver.php

```diff
@@ -119,20 +125,19 @@ public function get($id)
     {
         try {
             $statement = 'SELECT kv_value, kv_time FROM kv_store WHERE kv_id = ?';
-            $statement = $this->sqlPeristence->getPlatForm()->limitStatement($statement, 1);
-            $sessionValue = $this->sqlPeristence->query($statement, [$id]);
+            $statement = $this->sqlPersistence->getPlatForm()->limitStatement($statement, 1);
+            $sessionValue = $this->sqlPersistence->query($statement, [$id]);
             while ($row = $sessionValue->fetch()) {
-                if ($row['kv_time'] == 0 || $row['kv_time'] >= time()) {
-                    return filter_var($row['kv_value'], FILTER_VALIDATE_INT) !== false
-                        ? (int)$row['kv_value']
-                        : base64_decode($row['kv_value']);
+                if ($row["kv_time"] == 0 || $row["kv_time"] >= time()) {
+                    return base64_decode($row["kv_value"]);
```

Comment:
please see https://github.com/oat-sa/generis/pull/732/files#r375290983

Reply:
I'm keeping this part to be able to restore old int value, new ones will be base64encoded